Sources/TuistSupport/Utils/FileHandler.swift

```diff
@@ -63,6 +63,7 @@ public protocol FileHandling: AnyObject {
     func locateDirectory(_ path: String, traversingFrom from: AbsolutePath) -> AbsolutePath?
     func glob(_ path: AbsolutePath, glob: String) -> [AbsolutePath]
     func throwingGlob(_ path: AbsolutePath, glob: String) throws -> [AbsolutePath]
+    func throwingGlobExcludingOpaqueDirectories(_ path: AbsolutePath, glob: String) throws -> [AbsolutePath]
```

Comment:
maybe we could document this function with also defining what we mean by "opaque directory"?

Reply:
Will add documentation and move as extension of AbsolutePath.